components/Buttons/tests/unit/ButtonsTests.m

```diff
@@ -521,6 +527,35 @@ - (void)testTitleFontForStateFallbackBehavior {
   }
 }
 
+/** Test dynamic type responds to @c UIContentSizeCategoryDidChangeNotification. */
+- (void)testDynamicTypeBehavior {
+  if (@available(iOS 10.0, *)) {
+    // Given
+    TestButton *fakeButton = [[TestButton alloc] init];
+    fakeButton.mdc_adjustsFontForContentSizeCategory = YES;
+    fakeButton.mdc_legacyFontScaling = NO;
+    UIFont *titleFont = [UIFont systemFontOfSize:20];
+    MDCFontScaler *fontScaler = [MDCFontScaler scalerForMaterialTextStyle:MDCTextStyleButton];
+    titleFont = [fontScaler scaledFontWithFont:titleFont];
+    titleFont = [titleFont mdc_scaledFontAtDefaultSize];
```

Comment:
Tip: I think it would be easier to read if this font were taken from a DT-supporting typography scheme. Having to understand the MDCFontScaler code (which is meant to be an implementation detail) makes the test harder to read.

Reply:
This has been removed in favor of Snapshot test.